### PR TITLE
Instructions on how to install binaryen

### PR DESCRIPTION
**Pull request by MrishoLukamba**
From the docs there were no clear instructions on installing binaryen for newbies, I think this PR will add clear way to install binaryen

### CODE REVIEW

File: docs/getting-started/setup.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ The first tool we will be installing is [`cargo-contract`](https://github.com/pa
 
 As a pre-requisite for the tool you need to install the [binaryen](https://github.com/WebAssembly/binaryen) package, which is used to optimize the WebAssembly bytecode of the contract.
 
+For example, on Ubuntu or Debian, run the following command:
```
Comment (HCastano) [inline docs/getting-started/setup.md:19]:
```suggestion
To install it on Ubuntu or Debian, run the following command:
```

File: docs/getting-started/setup.md [hunk 2]
```diff
@@ -16,6 +16,17 @@ The first tool we will be installing is [`cargo-contract`](https://github.com/pa
...
+
+```bash
+sudo apt install binaryen
+```
+For macOs run
+```bash
+brew install binaryen
+```
+For other operating systems, download from [WebAssembly releases](https://github.com/WebAssembly/binaryen/releases).
```
Comment (HCastano) [inline docs/getting-started/setup.md:28]:
```suggestion

For installation on macOS run:

```bash
brew install binaryen
```

For other operating systems, download from [WebAssembly releases](https://github.com/WebAssembly/binaryen/releases).
```

File: docs/getting-started/setup.md [hunk 3]
```diff
@@ -16,6 +16,17 @@ The first tool we will be installing is [`cargo-contract`](https://github.com/pa
 
 As a pre-requisite for the tool you need to install the [binaryen](https://github.com/WebAssembly/binaryen) package, which is used to optimize the WebAssembly bytecode of the contract.
 
+For example, on Ubuntu or Debian, run the following command:
+
+```bash
+sudo apt install binaryen
+```
```
Comment (HCastano) [inline docs/getting-started/setup.md:23]: The latest stable [binaryen release is `v99`](https://tracker.debian.org/pkg/binaryen) which is too old for us to use, so this actually doesn't work at the moment. However, it is still a valid way of installing `wasm-opt`, so we can keep it but can you add a [caution admonition](https://docusaurus.io/docs/markdown-features/admonitions
) right below the installation instruction mentioning that users should manually download an up to date binary instead?

**Review by HCastano — COMMENTED** [review]
Thanks for the PR so far!

**Review by ascjones — COMMENTED** [review]
`cargo-contract` will now include `wasm-opt` with https://github.com/paritytech/cargo-contract/pull/766, so these instructions will become redundant

**Comment by cmichi** [comment]
@MrishoLukamba Thank you for the PR! As Andrew mentioned above we were fortunately able to remove `binaryen` as a third-party dependency by now, so the installation instructions are no longer necessary and I'm closing the PR hence.

**Pull request closed without merging**